onebusaway-android/src/main/res/values/strings.xml

```diff
@@ -735,4 +736,247 @@
+    <string name="step_by_step_non_transit_roundabout_ordinal_fourth">fourth</string>
+    <string name="step_by_step_non_transit_roundabout_ordinal_fifth">fifth</string>
+    <string name="step_by_step_non_transit_roundabout_ordinal_sixth">sixth</string>
+    <string name="step_by_step_non_transit_roundabout_ordinal_seventh">seventh</string>
+    <string name="step_by_step_non_transit_roundabout_ordinal_eighth">eighth</string>
+    <string name="step_by_step_non_transit_roundabout_ordinal_ninth">ninth</string>
+    <string name="step_by_step_non_transit_roundabout_ordinal_tenth">tenth</string>
+
+    <!-- To complement the step_by_step_non_transit_roundabout_number for a bigger ordinal -->
+    <string name="step_by_step_non_transit_roundabout_number">number</string>
+
+    <!-- After the ordinal -->
+    <string name="step_by_step_non_transit_roundabout_exit">exit</string>
+
+    <!-- Street type -->
+    <string name="street_type_bike_path">bike path</string>
+    <string name="street_type_open_area">open area</string>
+    <string name="street_type_path">path</string>
+    <string name="street_type_bridleway">bridleway</string>
+    <string name="street_type_footpath">footpath</string>
+    <string name="street_type_platform">platform</string>
+    <string name="street_type_footbridge">footbridge</string>
+    <string name="street_type_underpass">underpass</string>
+    <string name="street_type_road">road</string>
+    <string name="street_type_ramp">ramp</string>
+    <string name="street_type_link">link</string>
+    <string name="street_type_service_road">service road</string>
+    <string name="street_type_alley">alley</string>
+    <string name="street_type_parking_aisle">parking aisle</string>
+    <string name="street_type_byway">byway</string>
+    <string name="street_type_track">track</string>
+    <string name="street_type_sidewalk">sidewalk</string>
+    <string name="street_type_steps">steps</string>
+
+    <!-- Notifications specific -->
+
+    <!-- Button -->
+    <string name="notification_disable_updates_button">Cancel updates for this trip</string>
+
+    <!-- Toast -->
+    <string name="notification_disable_updates_info">Updates cancelled</string>
+
+    <!-- Text -->
+    <string name="notification_stop_name_conector">at</string>
+    <string name="notification_two_delays_connector">and</string>
+    <string name="notification_origin">origin</string>
+    <string name="notification_destination">destination</string>
+
+    <!-- Obtaining real time updates -->
+    <string name="toast_realtime_updates_fail">Realtime updates not available for this trip, times might be outdated</string>
+
+    <!-- Units -->
+    <string name="time_short_hours">h</string>
+    <string name="time_short_minutes">m</string>
+    <string name="time_short_seconds">s</string>
+    <string name="time_long_hours">h</string>
+    <string name="time_long_minutes">min</string>
```

Comment:
For time units, we already have:

    <string name="minutes_abbreviation">min</string>
    <string name="minutes_full">minutes</string>

Unless there is a reason not to, let's use these to be consistent throughout OBA.